src/app/services/cross-table.service.spec.ts

```diff
@@ -198,134 +198,4 @@ describe('CrossTableService', () => {
     expect(spySet).toHaveBeenCalled();
   });
 
-  it('should correctly create cross constraints for a concept', () => {
```

Comment:
@beaugogh Why did you delete these tests?

Reply:
Covered by `cross-table.integration.spec`, right?

Reply:
yes, should be covered by the integration tests, also, for some reason, the tests failed at these tests frequently, when I removed them, the tests passed